composer.json

```diff
@@ -25,7 +25,7 @@
         "hhvm/hsl": "^4.25",
         "hhvm/hsl-experimental": "^4.58.0rc1",
         "hhvm/hsl-io": "^0.2.0",
-        "hhvm/hhvm-autoload": "^2.0.4|^3.0",
```

Comment:
Tried running tests with this setting and I could not get it working, hhvm-autoload was 3.1 for HHVM 4.72. Not sure if this change is required, just wanted to add it for review.

Reply:
These changes should be unneeded, assuming you're letting composer install them - these constraints should always the latest version. If it's using stale versions, try deleting vendor/ and composer.lock, and trying again